modules/core/src/lib/deck.ts

```diff
@@ -809,6 +807,7 @@ export default class Deck {
       onInitialize: context => this._setDevice(context.device),
 
       onRender: this._onRenderFrame.bind(this),
+      // @ts-expect-error typing mismatch
```

Comment:
Can comment be a little more specific?

Reply:
AnimationLoop does not accept `onError: null`